geometry_msgs/Pose2D base_pose
float32[] joint_angles
